### DISCUSSION TITLE
Increase the number of buttons

### DISCUSSION THREAD

**Discussion by errFenotS** [AntiMicroX/antimicrox#692] [Ideas]
My Game Controller is Fiydigi Vader 2 .  It has four back buttons and two additional front buttons ("C" "Z") . 
   In the AntiMicoX 3.32,  "Controller Mapping" didn't reserve their place .  They can map Strings in a preemptive way ( Should I describe it this way,  I have just contacted AntiMicroX,  and I do not know the underlying logic,  String, API or SDL2 ) .   
  "Options"--> "property"-->"buttons" can detect these 6 buttons,  but cannot respond ( blue ) unless in a preemptive way .
   The main page can also show them button 16～ button 21,  as if they are eagerly waiting to map a keyboard letter。
   How should I read the source code and modify it，detect --> respond --> map a letter。

**Comment by pktiuk** [comment]
Answer to this question can be found in related issue: https://github.com/AntiMicroX/antimicrox/issues/694
